src/frogger.cpp

```diff
@@ -48,21 +58,28 @@ leapfrog::TensorMap1<int> get_age_groups_hiv_span(const Rcpp::List projection_pa
 Rcpp::List run_base_model(const Rcpp::List data,
                           const Rcpp::List projection_parameters,
                           SEXP sim_years,
+                          SEXP hiv_steps_per_year,
```

Comment:
Is there a way we can set a default value of 10? I tried this but ran into issues casting to SEXP, not sure what the proper way to do this is?